### ISSUE TITLE
Custom color overrides are not always dynamically detected

### ISSUE THREAD

**Issue by daeavelwyn** [jtackaberry/reaticulate#106]
Custom color doesn't work for me with the reapack version for now 0.4.4. So I try manually installing the 0.4.5 and custom colors now work as they should.

here is the begin of my Reaticulate.reabank file :

```
// Reabank file for Reaticulate - http://reaticulate.com/
//
// This file is a specially formatted Reabank file that is notated for use by Reaticulate.
// Specification can be found here:
//
//      http://reaticulate.com/reabank.html
//

//----------------------------------------------------------------------------

//! colors=art1=#FCEF54,art2=#1E88BA,art3=#97277C,art4=#9FD500,art5=#D55000


//////////////////////////////
//		BERLIN WODWINDS		//
//////////////////////////////

//! g="Orchestral Tools/Berlin WoodWinds" n="Piccolo"
//! m=""
Bank 2 1 OT-BW - Piccolo
```

**Comment by daeavelwyn** [comment]
update 0.4.4 to 0.4.5 version.

**Comment by jtackaberry** [comment]
Hm, 0.4.5 is included in the [release ReaPack](https://reaticulate.com/release.xml), so you shouldn't have needed a manual installation.

Playing with it now, I see the changes aren't getting picked up after clicking the reload icon in Reaticulate's toolbar as expected, and instead require restarting Reaticulate.  Could it be that the update to 0.4.5 was a red herring and it worked just because of the required restart?  Or had you tried restarting Reaticulate before?

**Comment by jtackaberry** [comment]
Ah, I see the problem. You must be using the prerelease ReaPack?  I seem to have neglected to include 0.4.5 there.  I'll fix that straightaway.

0.4.5 didn't include any changes to this area of the code, so I think the original bug you ran into still exists, and in fact I was able to reproduce it as well, although it's intermittent.

I'm going to reopen this to track it.

**Comment by jtackaberry** [comment]
Calling this fixed in 0.5.x.  Made several incidental code changes in this area, and I'm not able to reproduce it.  Custom color names seem to be working reliably.
